pkg/cmd/cli/cmd/edit.go

```diff
@@ -123,17 +136,29 @@ func RunEdit(fullName string, f *clientcmd.Factory, out io.Writer, cmd *cobra.Co
 	}
 
 	r := b.Flatten().Do()
-	infos, err := r.Infos()
+	o.infos, err = r.Infos()
 	if err != nil {
 		return err
 	}
 
-	defaultVersion := cmdutil.OutputVersion(cmd, clientConfig.Version)
-	results := editResults{}
+	o.version = cmdutil.OutputVersion(cmd, clientConfig.Version)
+	o.results = editResults{}
+	return nil
+}
+
+/*
+func (o *EditOptions) Validate(args []string) error {
```

Comment:
In here you need to validate all the Options that you completed above. See for example https://github.com/openshift/origin/blob/255307e708defdbdb66fc421cc3d45cce375a209/pkg/cmd/cli/cmd/rollback.go#L150